ROC-2974: Remove app/* alias and update all imports

ellie-harrison opened the pull request:
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-2974

### Change description
Removed TS alias "app/*" and updated all imports to follow a consistent style.


### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

timja-kainos commented on tsconfig.json:
```diff
@@ -12,91 +12,101 @@
-        "app/claims/*"
+        "main/app/claims/*"
+      ],
+      "app-common/*": [
```
what do you think about using either `common` or `shared` instead of `app-common` I'm not a big fan of the combined name?

ellie-harrison replied:
`common` would have conflicts with another alias, but `shared` would certainly be a better name.

geek-soft replied:
Happy with `shared` to start with. In the future we would like to remove app directory so the need for having both `common` and `shared` might disappear.

ellie-harrison commented:
I have updated the `app-common` to be `common` (as it is the most commonly used of the two directories called common) and the conflicting `common` to be `shared`.

Additionally I have added 2 new aliases for the files that were imported using 'main' to tidy those up a bit.
